modules/ossm-rn-fixed-issues.adoc

```diff
@@ -14,7 +14,30 @@ Provide the following info for each issue if possible:
 *Result* - How has the behavior changed as a result? Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
-The following issue has been resolved in the current release:
+The following issues have been resolved in the current release:
+
+* https://issues.redhat.com/browse/OSSM-1397[OSSM-1397] Previously, if you removed the `maistra.io/member-of` label from a namespace, the {SMProductShortName} Operator did not not automatically reapply the label to the namespace. As a result, sidecar injection did not work in the namespace.
++
+The Operator would reapply the label to the namespace when you made changes to the `ServiceMeshMember` object, which triggered the reconciliation of this member object.
++
+Now, any change to the namespace also triggers the member object reconciliation.
+
+* https://issues.redhat.com/browse/OSSM-4851[OSSM-4851] Previously, when you attempted to create a new pod in a namespace that was part of a service mesh, the {SMProductShortName} Operator failed to create the pod. The Operator generated the following error message:
++
+`failed to run injection template: failed parsing generated injected YAML (check Istio sidecar injector configuration): unmarshal patched pod: json: cannot unmarshal string into Go struct field SecurityContext.spec.containers.securityContext.runAsGroup of type int64`
++
+Each time you create a new pod associated with a service mesh, the Operator creates a sidecar proxy using YAML template files. In the template files, if the `unAsGroup`, `runAsUser`, or `fsGroup` parameters were set to `nil` because of the current configuration that needed to be applied to the proxy, then the Operator could not deploy a new pod.  
++
+The template files are used to create the `Istio-sidecar-injector-<SMCP-Name>` ConfigMap, located in the service mesh namespace. For example, the `istio-system` namespace.
```

Comment:
```suggestion
The template files are used to create the `istio-sidecar-injector-<SMCP-Name>` ConfigMap, located in the service mesh namespace. For example, the `istio-system` namespace.
```

Reply:
Why are we talking about template files? If these templates from the Helm charts, then we shouldn't talk about them, as they are an implementation detail that we never talk about elsewhere.

Reply:
@luksa do you think saying something like "A chart template is a collection of files that describe a related set of Kubernetes resources." is reasonable?

Reply:
No, I wouldn't mention charts or templates at all. Users never deal with them.

Reply:
@luksa so you think we can remove line 31, correct?

Reply:
Line 31 and others as well.

Reply:
Removed the entry per group discussion during review.